test/Unit_Tests_Objects/active_time_ranges_test.dart

```diff
@@ -0,0 +1,30 @@
+/// Copyright 2020 Google LLC
```

Comment:
test/Unit_Tests_Objects/active_time_ranges_test.dart should be test/unit_tests/objects/active_time_ranges_test.dart

Reply:
Done.

Reply:
Done.